index.Rmd

```diff
@@ -261,23 +262,18 @@ as(MNDWI, "SpatialPixelsDataFrame") %>%
 
 I loaded required packages
 
-```{r, message=F, warning=F}
-library(threejs)
-```
 
 ##Results2
 I firstly prepared and loaded the data and then with the package called "streejs" I plot the exact locations of landing and mining sites on the Martian global map. I created a cvs file where I stored the latitudes and logitude of the future landing and mining sites selested and call that file in R.
 
 
 ```{r}
-library(threejs)
-library(csvread)
```

Comment:
Best practice: List all libraries at the beginning of code. See comment in your original libraries calls.